app-crypt/veracrypt/files/veracrypt.init

```diff
@@ -0,0 +1,33 @@
+#!/sbin/openrc-run
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+depend() {
+	after localmount
+	if [ -e /lib/librc.so ]; then
+		need device-mapper
+	fi
+}
+
+start() {
```

Comment:
Per https://www.gentoo.org/support/news-items/2012-02-14-baselayout-1-deprecation.html, baselayout-1 is long gone, we don't need this check here.